batch7crud-roll4/src/main/webapp/resources/css/reset.css

```diff
@@ -0,0 +1,20 @@
+* {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this usage of the universal selector as key part [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=css%3Auniversal-selector)